src/components/overlays/teleport-container.ts

```diff
@@ -0,0 +1,23 @@
+import Vue from 'vue';
+
+export default Vue.extend({
```

Comment:
small question, doesn't defineComponent(..) work the same way here?

Reply:
let me try it, initially used setup, and it was giving me a different value on import.

Reply:
`defineComponent` is giving me the same vue instance, what is needed here is the Vue constructor.